t/spec/test_oidc.lua

```diff
@@ -0,0 +1,36 @@
+describe("OIDC auth plugin", function()
```

Comment:
We use a different way to test. You can check it out in https://github.com/apache/apisix/blob/master/docs/en/latest/internal/testing-framework.md

The test code needs to be added in https://github.com/apache/apisix/blob/master/t/plugin/openid-connect2.t